app/src/main/kotlin/com/aliucord/manager/ui/component/LoadFailure.kt

```diff
@@ -0,0 +1,34 @@
+package com.aliucord.manager.ui.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun LoadFailure(onRetry: () -> Unit) {
+    Column(
+        verticalArrangement = Arrangement.spacedBy(8.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.fillMaxWidth()
```

Comment:
since when

Reply:
Since forever, nearly every material 3 composable has it as the first parameter because its a universal parameter. it makes sense to have it the same on every single one

Reply:
That's still a convention by the compose authors for arguments, it's not really an official thing. I put it last as a paramater because it's less lines changed if you want to add an extra modifier on a new line

Reply:
you are insane

Reply:
nop
nobody puts modifier first